dotcom-rendering/src/web/components/SignInGate/SignInGate.stories.tsx

```diff
@@ -20,26 +23,13 @@ export const mainStandalone = () => {
 				signInUrl="https://profile.theguardian.com/"
 				dismissGate={() => {}}
 				ophanComponentId="test"
+				personaliseSignInAfterCheckoutSwitch={false}
 			/>
 		</Section>
 	);
 };
 mainStandalone.story = { name: 'main_standalone' };
 
-export const mainStandaloneComment = () => {
```

Comment:
Is there a reason for removing these stories?

Reply:
I understand this was hangover from a previous iteration of the sign in gate. I think this was something @vlbee did when we were pairing, so perhaps she could say more?

Reply:
I could see no difference at all in the stories code or component compared to the mainStandalone gate for what was being rendered.